src/os/bluestore/BlueStore.h

```diff
@@ -735,6 +735,18 @@ class BlueStore : public ObjectStore,
   };
   typedef boost::intrusive_ptr<Onode> OnodeRef;
 
+  //Made a specific class to be able to verify GC logics in Unit Tests
+  struct GarbageCollector {
```

Comment:
I think it's simpler to just make this a method of ExtentMap, which already includes several unit tests in test_bluestore_types.cc